WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -13,16 +13,27 @@
                  EnableNext="True" NextContent="Done"
                  EnableCancel="{Binding EnableCancel}"
                  EnableBack="{Binding EnableBack}">
-    <StackPanel Classes="settingsLayout">
-      <DockPanel IsVisible="{Binding IsHardwareWallet}">
-        <TextBlock Text="PSBT workflow" />
-        <ToggleSwitch IsChecked="{Binding PreferPsbtWorkflow}" />
-      </DockPanel>
+    <StackPanel Spacing="20">
+      <StackPanel Classes="settingsLayout">
+        <DockPanel IsVisible="{Binding IsHardwareWallet}">
+          <TextBlock Text="PSBT workflow" />
+          <ToggleSwitch IsChecked="{Binding PreferPsbtWorkflow}" />
+        </DockPanel>
 
-      <DockPanel IsVisible="{Binding !IsWatchOnly}">
-        <TextBlock Text="Auto CoinJoin" />
-        <ToggleSwitch IsChecked="{Binding AutoCoinJoin}" />
-      </DockPanel>
+        <DockPanel IsVisible="{Binding !IsWatchOnly}">
+          <TextBlock Text="Auto CoinJoin" />
+          <ToggleSwitch IsChecked="{Binding AutoCoinJoin}" />
+        </DockPanel>
+      </StackPanel>
+      <StackPanel Spacing="20">
+        <TextBlock Classes="h6" Text="Have you checked your wallet recovery seed words?" />
+        <c:InfoMessage>
+          <DockPanel>
+            <Button Margin="50 0" Command="{Binding VerifyRecoveryPhraseCommand}" Content="Verify Recovery Seed Words" DockPanel.Dock="Right" />
+            <TextBlock Text="It is recommended that you verify your wallet recovery seed words every 6 months." />
```

Comment:
Should `Recovery Words` start with capital letters? And remove the word `seed`.

Why 6 months? Where did this recommendation come from?

Reply:
Saw it in another wallet, seems sensible advice, gives some context that its something you should do.

Reply:
corrected some of the gramatical issues.